plugins/nf-k8s/src/main/nextflow/k8s/model/PodSpecBuilder.groovy

```diff
@@ -599,6 +607,9 @@ class PodSpecBuilder {
         if( ttlSecondsAfterFinished != null )
             spec.ttlSecondsAfterFinished = ttlSecondsAfterFinished
 
+        if( runtimeClassName )
+            spec.runtimeClassName = runtimeClassName
```

Comment:
Can the runtime class be specified in the Job spec? I assume the job would pick it up from the pod spec

Reply:
No, only in the pod spec, also when included in a job, but not in the job spec itself. Good question, I checked that up before sending the patch :)